view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js

```diff
@@ -438,6 +438,28 @@ define(
             getCode: function() {
                 return window.checkoutConfig.payment.adyenCc.methodCode;
             },
+
+            getTitle: function () {
+                const paymentMethodsObservable = adyenPaymentService.getPaymentMethods();
+
+                if (
+                    typeof paymentMethodsObservable === 'function' &&
+                    paymentMethodsObservable() &&
+                    paymentMethodsObservable().paymentMethodsResponse &&
+                    Array.isArray(paymentMethodsObservable().paymentMethodsResponse.paymentMethods)
+                ) {
```

Comment:
I think there can be a better approach here in order to get the data from that function cause this way you are running it three times in order to get the paymentMethods